stats/justfile

```diff
@@ -25,7 +25,7 @@ test *args:
     cargo test {{args}} -- --include-ignored
 
 test-with-db *args:
-    just db-port="{{test-db-port}}" db-name="" docker-name="{{docker-name}}-test" start-postgres
+    -just db-port="{{test-db-port}}" db-name="" docker-name="{{docker-name}}-test" start-postgres
```

Comment:
Should `-` not be here?

Reply:
you can put minus before command to ignore exit code and continue executing commands

Reply:
https://github.com/casey/just/tree/master?tab=readme-ov-file#ignoring-errors